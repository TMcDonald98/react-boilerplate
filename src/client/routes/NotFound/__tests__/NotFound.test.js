import * as React from 'react';
import {
  NotFound,
} from '../NotFound';

describe('components/NotFound', () => {
  describe('render', () => {
    it('should render without error', () => {
      const result = shallow(<NotFound />);

      expect(result).toMatchSnapshot('NotFound');
    });
  });
});
