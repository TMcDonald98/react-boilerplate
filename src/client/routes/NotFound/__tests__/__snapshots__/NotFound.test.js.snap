// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/NotFound render should render without error: NotFound 1`] = `
<div
  className="not-found-container"
>
  <h1
    className="not-found-title"
  >
    404
  </h1>
  <p>
    Page not found.
  </p>
</div>
`;
